core/src/test/scala/org/apache/spark/deploy/JsonProtocolSuite.scala

```diff
@@ -25,24 +25,11 @@ import org.json4s._
 import org.json4s.jackson.JsonMethods
 import org.scalatest.FunSuite
 
-import org.apache.spark.deploy.DeployMessages.{MasterStateResponse, WorkerStateResponse}
-import org.apache.spark.deploy.master.{ApplicationInfo, DriverInfo, RecoveryState, WorkerInfo}
+import org.apache.spark.{JsonTestUtils, SparkConf}
+import org.apache.spark.deploy.DeployMessages.WorkerStateResponse
 import org.apache.spark.deploy.worker.{DriverRunner, ExecutorRunner}
-import org.apache.spark.SparkConf
 
-class JsonProtocolSuite extends FunSuite {
-
-  test("writeApplicationInfo") {
```

Comment:
thanks for pointing this out -- these changes actually should not have been checked in, this functionality has not been replaced elsewhere.

I should also probably clarify what the other (intended) changes to do backwards-compatibility: The previous `XyzPage` classes had sporadic implementations of `renderJson`.  Other than `Master` and `Worker`, those methods have been removed, with new functionality available from the new routes.  I made no attempt to keep the json the same (its almost certainly different in every case).  I thought there wasn't really a guarantee of backwards-compatibility given how incomplete the previous implementation was, and figured things would be clearer if we just completely removed them.  If we do want to keep those old apis active, its easy enough to drop them back in.

I have not updated the way json is returned for the standalone cluster, specifically in `MasterPage` & `WorkerPage`.  I figure that is independent enough it can go in another PR if we really want to update it, but if you have a strong preference I can update them here as well.

(In the meantime, I will undo my changes to `MasterPage` and `JsonProtocolSuite` since right now that functionality is just missing.)